src/components/console/utils/safe-fetch-hook.ts

```diff
@@ -9,5 +9,5 @@ export const useSafeFetch = () => {
   }, []);
 
   return (url) =>
-    consoleFetchJSON(url, 'get', { signal: controller.current.signal as AbortSignal });
+    consoleFetchJSON(url, 'get', { signal: controller.current?.signal as AbortSignal });
```

Comment:
@zhuje @kyoto For some reason this is undefined so I had to add an optional chain here. Do you have any idea why it is happening?